toqito/nonlocal_games/nonlocal_game.py

```diff
@@ -5,12 +5,76 @@
 
 import cvxpy
 import numpy as np
+import numba as nb
 
 from toqito.helper import update_odometer
 from toqito.matrix_ops import tensor
 from toqito.rand import random_povm
 from toqito.state_opt.npa_hierarchy import npa_constraints
 
+from toqito.nonlocal_games.binary_constraint_system_game import check_perfect_commuting_strategy
+from toqito.matrix_ops.tensor_unravel import tensor_unravel
+
+
+@nb.njit
+def _fast_classical_value(pred_mat: np.ndarray, num_b_out: int, num_b_in: int, pow_arr: np.ndarray) -> float:
```

Comment:
Out of curiosity, is there any reason that you can't just put the internals of this function within the `classical_value` function instead? That is, I don't think we need a separate `_fast_classical_value` function--we can just assume that `classical_value` does the calculation in a fast way. Does that make sense?